app/Console/Kernel.php

```diff
@@ -24,11 +24,9 @@ class Kernel extends ConsoleKernel {
 
     /**
      * Define the application's command schedule.
-     *
-     * @param  \Illuminate\Console\Scheduling\Schedule  $schedule
-     * @return void
      */
-    protected function schedule(Schedule $schedule) {
+    protected function schedule(Schedule $schedule): void {
+        $schedule->command('cache:prune-stale-tags')->hourly();
```

Comment:
If this is the new standard, should we try and change all the functions we have (add types to all of them)? Or would you prefer to just follow this format from here on out? Either work for me.

Reply:
This is the new standard. I'd prefer to modify things as we go since this isn't a breaking change.